lib/recurly/client.php

```diff
@@ -49,6 +49,7 @@ class Recurly_Client
   const PATH_BILLING_INFO = '/billing_info';
   const PATH_COUPON = '/coupon';
   const PATH_COUPON_REDEMPTION = '/redemption';
+  const PATH_COUPON_REDEMPTIONS = '/redemptions';
```

Comment:
It's not actually being called. Try running `Recurly_CouponRedemptionList::get();` and you'll get a 404 back.